mitodl/open-discussions
# Move footer into sidebar

**Rebased** on #1082 

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #986 

#### What's this PR do?
Updates footer to match the mockup and moves it into the sidebar. The sidebar is changed to use flexbox and the footer is pushed to the bottom of the screen via flex-shrink

#### How should this be manually tested?
View the sidebar and verify that the footer sits at the bottom, looks good and the links work as before
